net/httpie-go/Portfile

```
@@ -0,0 +1,147 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/nojima/httpie-go v0.7.0
```

The format of `github.setup` is `<Go repo> <version> <version prefix> <version suffix>`:

```suggestion
go.setup            github.com/nojima/httpie-go 0.7.0 v
```